CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [v1.0.0-rc.4] - 2019-12-17
+
+### Changes
+- Update kooper to v0.8.0
+- Update kubernetes to v1.15.6
+- Add support for `hostNetwork` and `dnsPolicy` in Redis and Sentinel pods
```

Comment:
It would be great mention here the author

Reply:
indeed :)